package.json

```diff
@@ -3,28 +3,79 @@
+    "e2e": "yarn cypress install && yarn grafana-e2e run",
+    "e2e:update": "yarn cypress install && yarn grafana-e2e run --update-screenshots",
+    "lint": "eslint --cache --ignore-path ./.gitignore --ext .js,.jsx,.ts,.tsx .",
+    "lint:fix": "yarn lint --fix",
+    "server": "docker-compose up --build",
+    "sign": "npx --yes @grafana/sign-plugin",
     "spellcheck": "cspell -c cspell.config.json \"**/*.{ts,tsx,js,go,md,mdx,yml,yaml,json,scss,css}\"",
-    "build": "rm -rf dist && grafana-toolkit plugin:build && mage build:backend",
-    "test": "grafana-toolkit plugin:test",
-    "lint": "grafana-toolkit plugin:lint",
-    "dev": "rm -rf node_modules/@grafana/data/node_modules; grafana-toolkit plugin:dev",
-    "watch": "grafana-toolkit plugin:dev --watch"
+    "test": "jest --watch --onlyChanged",
+    "test:ci": "jest --passWithNoTests --maxWorkers 4",
+    "typecheck": "tsc --noEmit"
   },
   "repository": "github:grafana/iot-sitewise-datasource",
   "author": "Grafana Labs <team@grafana.com> (https://grafana.com)",
   "license": "Apache-2.0",
   "devDependencies": {
+    "@babel/core": "^7.16.7",
     "@grafana/aws-sdk": "^0.0.37",
-    "@grafana/data": "8.5.0",
-    "@grafana/runtime": "8.5.0",
-    "@grafana/toolkit": "8.5.0",
-    "@grafana/ui": "8.5.0",
-    "@types/lodash": "latest",
+    "@grafana/e2e": "9.2.5",
+    "@grafana/e2e-selectors": "9.2.5",
+    "@grafana/eslint-config": "^5.1.0",
+    "@grafana/tsconfig": "^1.2.0-rc1",
+    "@swc/core": "^1.2.144",
+    "@swc/helpers": "^0.4.12",
+    "@swc/jest": "^0.2.23",
+    "@testing-library/jest-dom": "^5.16.2",
+    "@testing-library/react": "^12.1.4",
+    "@types/glob": "^8.0.0",
+    "@types/jest": "^29.2.2",
+    "@types/lodash": "^4.14.188",
+    "@types/node": "^18.11.9",
+    "@typescript-eslint/eslint-plugin": "^5.42.1",
+    "@typescript-eslint/parser": "^5.42.1",
+    "copy-webpack-plugin": "^11.0.0",
     "cspell": "6.13.3",
+    "css-loader": "^6.7.1",
+    "eslint": "8.26.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-jsdoc": "^39.6.2",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "^7.26.1",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "eslint-webpack-plugin": "^3.1.1",
+    "fork-ts-checker-webpack-plugin": "^7.2.0",
+    "glob": "^8.0.3",
+    "identity-obj-proxy": "3.0.0",
+    "jest": "^29.3.1",
+    "jest-environment-jsdom": "^29.3.1",
     "lodash": "latest",
-    "tslib": "2.2.0"
+    "prettier": "^2.5.0",
+    "replace-in-file-webpack-plugin": "^1.0.6",
+    "sass": "1.56.1",
+    "sass-loader": "13.2.0",
+    "style-loader": "3.3.1",
+    "swc-loader": "^0.2.3",
+    "ts-node": "^10.5.0",
+    "tsconfig-paths": "^4.1.0",
+    "tslib": "2.2.0",
+    "typescript": "^4.4.0",
+    "webpack": "^5.69.1",
+    "webpack-cli": "^4.9.2",
+    "webpack-livereload-plugin": "^3.0.2"
   },
   "resolutions": {
     "rxjs": "6.6.3"
+  },
+  "dependencies": {
+    "@emotion/css": "^11.1.3",
```

Comment:
There's this sneaky thing you have to do which is move all the dependencies listed in .config/webpack/webpack.config.ts back to devDependencies, since we should be using main grafana's versions of them

Reply:
Oof, didn't see this. Will correct it!